packages/api/lambdas/db-indexer.js

```diff
@@ -157,11 +157,12 @@ function performIndex(indexFnName, esClient, data) {
 /**
  * Perform the delete operation for the given record
  *
- * @param {Object} esClient - ElasticSearch connection client
- * @param {string} type - type of record to index
- * @param {string} id - record ID
- * @param {string} parentId - ID of parent record
- * @returns {Promise<Object>} elasticsearch response
+ * @param {string} deleteFnName - Delete function name
+ * @param {Object} esClient     - Elasticsearch connection client
```

Comment:
I thought this was under Mark's needs integration test flag?

Reply:
As long as we have a test that was failing before the changes in this PR and is now passing, then we're good. It would actually be hard to do an integration test given the asychronous nature of ES updates

Reply:
I concur with @markdboyd

Reply:
I'm good too.  should have looked at this first, just saw it approved already